test/integration/patients/worklist/schedule.js

```diff
@@ -481,7 +481,6 @@ context('schedule page', function() {
 
     cy
       .get('[data-select-all-region]')
-      .find('button')
```

Comment:
The select all icon gets re-rendered frequently now and it was causing a lot of flake in these tests because the element was removed from the dom. Hence this update to just click on the parent.